roles/manage-aws-infra/templates/casl-aws-cformation.yml.j2

```diff
@@ -785,3 +856,10 @@ Resources:
         ResourceRecords:
         - !GetAtt ETCD{{ '%02d' % idx }}.PublicDnsName
       {% endfor -%}
+      {% for idx in range(0, aws_num_bastion| int) -%}
+      - Name: "bastion-{{ idx }}.{{ env_id }}.{{ dns_domain }}."
+        Type: CNAME
+        TTL: 300
+        ResourceRecords:
+        - !GetAtt BAST{{ '%02d' % idx }}.PublicDnsName
```

Comment:
Check that you are declaring the bastion objects as 'Bastion{{ '%02d' % idx }}', not 'BAST{{ '%02d' % idx }}'. Please change this.